internal/nightfallconfig/nightfall_config.go

```diff
@@ -44,6 +44,7 @@ var defaultNightfallConfig = &NightfallConfigFileStructure{
 
 // NightfallConfigFileStructure struct representation of nightfall config file
 type NightfallConfigFileStructure struct {
+	ConditionSetUUID   string                    `json:"conditionSetUUID"`
```

Comment:
Is there a reason this is not `uuid.UUID` instead of `string`?

Reply:
The OpenAPI generated ConditionSetUUID type is also a string (uuids aren't supported), so I didn't want to convert this into a `uuid.UUID` and then back into a string as the `ScanRequestV2` struct expects.

Reply:
Gotcha! Should we do any validation to make sure the string is a valid UUID?

Reply:
Yea good question - it might be a good idea to fail here rather than failing downstream (there's validation on the proto when making the request to DSG that will error if it's invalid). I'll add a check here too